express-notes-api/server.js

```diff
@@ -0,0 +1,116 @@
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+
+const app = express();
+
+const middle = express.json();
```

Comment:
This is a pretty generic name. We will be installing quite a few other middlewares over time, so you should get used to using more descriptive names. Also, you could avoid this local variable entirely if you inline `express.json()` at line 8.